model_analyzer/config/generate/perf_analyzer_config_generator.py

```diff
@@ -238,6 +240,17 @@ def _create_concurrency_list(self) -> List[int]:
                 self._cli_config.run_config_search_max_concurrency,
             )
 
+    def _create_periodic_concurrency_list(self) -> List[int]:
+        if self._model_parameters["periodic_concurrency"]:
+            return sorted(self._model_parameters["periodic_concurrency"])
+        elif self._cli_config.run_config_search_disable:
+            return [1]
```

Comment:
magic number?

Reply:
Replaced this (and the others) with their default min values.